be/src/storage/lake/tablet_manager.cpp

```diff
@@ -56,6 +60,50 @@ std::string TabletManager::txn_log_path(const std::string& group, const std::str
     }
 }
 
+std::string TabletManager::txn_log_cache_key(int64_t tablet_id, int64_t txn_id) {
+    return fmt::format("txn_{:016X}_{:016X}", tablet_id, txn_id);
+}
+
+static void tablet_meta_data_deleter(const CacheKey& key, void* value) {
```

Comment:
```suggestion
static void tablet_metadata_deleter(const CacheKey& key, void* value) {
```

Reply:
Why named deleter, delete seems ok

Reply:
Because there need an object used for cleanup. deleter is a noun and delete  means  an action